quickstarts/oma-qf/config.yml

```
@@ -0,0 +1,43 @@
+id: 
+name: customer-experience-quality-foundation 
```

We recently put in a change to update this field name to be more accurately called `slug`. No changes needed other than renaming the field!
```suggestion
slug: customer-experience-quality-foundation 
```